docs/UserGuide.md

```diff
@@ -517,6 +590,9 @@ unmarkc CINDEX
 
 Sets a customer's note.
 
+Do you have additional information to remember about your customer? 
+This command helps you to remember anything you want about your customer.
```

Comment:
This sounds a little repetitive, maybe you can say something along the lines of "Note that information down with the `setnotec` command!" or "This command lets you can note down that information!"